src/main.rs

```diff
@@ -217,7 +219,6 @@ fn run(args: Cli) -> Result<ExitCode> {
                 format!("Failed to read the custom CA bundle: {}", path.display())
             })?;
 
-            client = client.tls_built_in_root_certs(false);
```

Comment:
This has been removed in https://github.com/seanmonstar/reqwest/issues/2885

Reply:
Sounds like the solution is to switch from `add_root_certificate()` to `tls_certs_only()`?